test/core/linter/linter_test.py

```diff
@@ -655,3 +655,13 @@ def test_normalise_newlines():
     in_str = "SELECT\r\n foo\n FROM \r \n\r bar;"
     out_str = "SELECT\n foo\n FROM \n \n\n bar;"
     assert out_str == Linter._normalise_newlines(in_str)
+
+
+def test_lint_jinja_macro_exclude():
+    """Test linting when excluding macros with unknown tags."""
+    config_path = "test/fixtures/templater/jinja_exclude_macro_path/.sqlfluff"
+    config = FluffConfig.from_path(config_path)
+    linter = Linter(config=config)
+    sql_file_path = "test/fixtures/templater/jinja_exclude_macro_path/jinja.sql"
+    lint = linter.lint_path(sql_file_path)
+    assert lint.num_violations() == 0
```

Comment:
What's your thinking on having this test in addition to the _parse_ one? If the error would normally have been raised at the templating/parsing phase, I'm tempted to suggest removing this one as redundant.

Open to being persuaded though if I've missed something.

Reply:
Yes, you are right - I will remove it. I wanted to ensure it was being tested properly but it is a duplicate.